lib/validators/cat3_population_validator.rb

```diff
@@ -26,6 +26,7 @@ def validate(file, options = {})
           # Otherwise, check proportion population are reported correctly
           validate_populations(measure_id, pop_counts, options)
         end
+        validate_population_ids(document, options)
```

Comment:
This should be moved outside of the "measure" loop.  It currently runs for each measure in the file.  So if there are 47 measures, it runs 47 times.  It only needs to run once.